### PR TITLE
Report specs found as candidates as github issue

### PR DESCRIPTION
**Pull request by dontcallmedom**
per #71

no idea if this will actually work, and not really sure how to test it beyond deploying it?

### CODE REVIEW

**Comment by tidoust** [comment]
> no idea if this will actually work, and not really sure how to test it beyond deploying it?

One way to test it is to deploy it on a fork. But we can just as well deploy it here. At worst it just won't work.

**Comment by dontcallmedom** [comment]
It actually [worked](#77)!

File: .github/candidate-specs.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: New specs for review
+assigneeds: tidoust, dontcallmedom
```
Comment (tidoust) [inline .github/candidate-specs.md:3]: I suspect s/assigneeds/assignees will fix the fact that the bot created an issue with assigning it to us.

Comment (dontcallmedom) [reply .github/candidate-specs.md]: that was a feature, but oh well

**Pull request merged**